<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ddmMaxInOutNeighborFilterWidget</class>
 <widget class="QWidget" name="ddmMaxInOutNeighborFilterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>195</width>
    <height>307</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>155</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ddmCountyFilterWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <widget class="QComboBox" name="comboType">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>&lt;все&gt;</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Соседи 1-го порядка</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Соседни 2-го порядка</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listCounties"/>
     </item>
     <item>
      <widget class="QCheckBox" name="checkSameState">
       <property name="text">
        <string>В одном штате</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
